### DISCUSSION TITLE
impossible to pay for organization account

### DISCUSSION THREAD

**Discussion by kiri03** [opengisch/QField#3935] [Q&A]
I am trying to pay to register for an organization account and when I fill in all the information and try to make the payment with the card, a charge of €0.0 appears, so it is impossible for me to accept the payment from the bank service.
I've been trying for several days and today is the last day to have the discount so I don't know the correct way to register before tomorrow the 22nd.
I hope someone can help me.

Thanks

**Comment by suricactus** [comment]
Hey, @kiri03, can you please send us an email at beta-transition@qfield.cloud and share you username, so we can check everything is good for your organization? As the first payment happens at midnight today, it is normal when you add your card to see 0.00 EUR , because we didn't charge you anything, just verified your card.
